server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -1,6 +1,8 @@
-  
+
 [[crossdc-mode]]
-=== Cross-Datacenter Replication Mode
```

Comment:
"center Replication Mode" looks like typo. IMO the header should remain as is?

Reply:
Thanks, that was my mistake and I can fix it.